schema/monitoring_metric.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$udmi_version": "1.4.1",
+    "title": "Monitoring metric",
+    "description": "One metric",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "properties": {
+      "timestamp": {
```

Comment:
NB: There's an existing structure in common.json#entry that has many of the same fields and is commonly used in other parts of the system.  Having a uniform structure just means that it'll be easier for some tools to consume this in a more standardized way.  The format of that was something that could be easily consumer, e.g., CloudLogging.  It's used for status and log messages generated by devices and tools. Slightly different than a metric, but something to at least consider.